### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by madany01**
complete mul and dev functions.

### CODE REVIEW

File: calc.py [hunk 1]
```diff
@@ -11,4 +11,4 @@ def mul(a, b):
 
 
 def div(a, b):
-    pass
+    return a / b
```
Comment (a-madany) [inline calc.py:14]: what if b is zero

Comment (madany01) [reply calc.py:14]: I will fix it.

**Comment by a-madany** [comment]
could you add pow function ?

**Comment by madany01** [comment]
yes

File: calc.py [hunk 2]
```diff
@@ -14,3 +14,7 @@ def div(a, b):
     if b == 0:
         return None
     return a / b
+
+
+def pow(a, b):
+    return a ** b
```
Comment (a-madany) [inline calc.py:20]: not optimized

**Review by a-madany — CHANGES_REQUESTED** [review]

**Review by a-madany — APPROVED** [review]

**Pull request merged**